sidebars.js

```diff
@@ -91,7 +92,6 @@ module.exports = {
             'go-distributed-cron',
             'go-tracing',
             'go-search-apis',
-            'go-selectors',
```

Comment:
bumping up selectors in the sidebar to put them roughly nearer where they'd come up conceptually... what do you think?

or do we do this alphabetically at this point..

Reply:
I was thinking that eventually they will be ordered by level of most used -